wordpress-mobile/WordPress-iOS
# UL&S: Add TextFieldTableViewCell to SiteAddressViewController

Fixes # n/a
Ref PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/306

### To test:
1. `rake dependencies`
2. Build and run
3. Log in > "Or log in by entering your site address."
4. The SiteAddressViewController appears and you can interact with the textfield. (The Continue button doesn't do anything yet.)

### Design review:
The spacing between table cells in Zeplin isn't selectable. I can see there is extra space between the label and the textfield but I don't know how much. What's rendered above is a minimum height of 44 points for the 2 cells. How much top / bottom margin should be between the 2 items?

**Zeplin design**
<img width="351" alt="Screen Shot 2020-06-17 at 11 29 01 AM" src="https://user-images.githubusercontent.com/1062444/84924164-c7f7bb00-b08d-11ea-8a16-9d9b7672ce5a.png">

**iPhone 11 Pro screenshots**
<a href="https://user-images.githubusercontent.com/1062444/84951840-a52dcc80-b0b6-11ea-9768-dfa478c7a065.png"><img src="https://user-images.githubusercontent.com/1062444/84951840-a52dcc80-b0b6-11ea-9768-dfa478c7a065.png" width="350" /></a>

<a href="https://user-images.githubusercontent.com/1062444/84951886-b8d93300-b0b6-11ea-9d80-3abfb0b8329e.png"><img src="https://user-images.githubusercontent.com/1062444/84951886-b8d93300-b0b6-11ea-9d80-3abfb0b8329e.png" width="350" /></a>


PR submission checklist:
